sympy/functions/elementary/hyperbolic.py

```diff
@@ -877,6 +878,35 @@ def _eval_as_leading_term(self, x, cdir=0):
         else:
             return self.func(arg)
 
+    def _eval_expand_trig(self, **hints):
+        arg = self.args[0]
+        x = None
+        if arg.is_Add:
+            from sympy import symmetric_poly
+            n = len(arg.args)
+            CX = []
+            for x in arg.args:
+                cx = coth(x, evaluate=False)._eval_expand_trig()
+                CX.append(cx)
+
+            Yg = numbered_symbols('Y')
+            Y = [next(Yg) for i in range(n)]
```

Comment:
```suggestion
            Y = symbols('Y:%s' % n)
```

Reply:
It should be possible to use `CX` instead of `Y`.